src/data/finset/lattice.lean

```diff
@@ -142,6 +142,21 @@ begin
     exact ⟨le_trans hay hyz, le_trans hsx_sup hxz⟩, },
 end
 
+-- If we acquire sublattices
+-- the hypotheses should be reformulated as `s : subsemilattice_inf_bot`
+lemma finset.sup_mem {α : Type*} [semilattice_sup_bot α]
```

Comment:
```suggestion
lemma sup_mem
```
I think you're already in a suitable namespace and have suitable assumptions

Reply:
Thanks!